README.md

```diff
@@ -26,6 +26,7 @@ You will need the following tools:
 - [CMake](https://cmake.org/)
 
 OS-specific instructions for installing these prerequisites can be found in the [Wiki](https://github.com/XboxDev/nxdk/wiki/Install-the-Prerequisites)
+Note that homebrew software build with nxdk won't work with [Cxbx-Reloaded](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded) because they don't support nxdks API.
```

Comment:
I feel like this isn't really useful here?

It's burried in other information and part of "Getting Started" with zero-context. Some people might not even know what Cxbx-Reloaded is.

I feel this should be integrated in the wiki instead; in fact, the wiki incorrectly *mentions* Cxbx-Reloaded as functional solution for nxdk.

I'm not sure *where* to put this kind of information in the wiki though.

Reply:
ftr, was based on a misunderstanding. The mentioning of cxbx-r in the wiki got removed.